doc/gambit.txi

```diff
@@ -13426,7 +13423,26 @@ $ @b{./foo 1 2 "3 4"}
 
 @end deffn
 
-@node Environment variables, Measuring time, Command line arguments, Host environment
+@deffn procedure executable-path
+
+This procedure returns a newly allocated string representing a
```

Comment:
I think we should avoid saying that the returned string is newly allocated, because I can imagine a different implementation (possibly a "better" implementation in terms of memory usage) where the string is not newly allocated.